src/list/ListItemAccordion.tsx

```diff
@@ -10,10 +10,12 @@ export type ListItemAccordionProps = ListItemProps & {
   icon?: IconNode;
   expandIcon?: IconNode;
   content?: React.ReactNode;
-  noAnimation?: boolean;
   noRotation?: boolean;
   noIcon?: boolean;
-  animationDuration?: number;
+  animation?: {
+    type?: 'timing' | 'spring';
+    duration?: number;
+  };
```

Comment:
`useNativeDriver` can only be false cuz height is not supported by native drivers

Reply:
type of `animation` should be `boolean | {animation_object}`
Boolean can take only false which means the animation should be disabled.

Is there any other way around?

Reply:
It didn't look nice without animation I guess, will become a simple hide and show toggle

Reply:
Done